new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/primitive/Installer.java

```diff
@@ -5,24 +5,42 @@
 
 import java.io.File;
 
+import static oracle.weblogic.kubernetes.actions.ActionConstants.DOWNLOAD_DIR;
+import static oracle.weblogic.kubernetes.actions.ActionConstants.IMAGE_TOOL;
+import static oracle.weblogic.kubernetes.actions.ActionConstants.WORK_DIR;
+import static oracle.weblogic.kubernetes.actions.impl.primitive.Command.defaultCommandParams;
 import static oracle.weblogic.kubernetes.extensions.LoggedTest.logger;
+import static oracle.weblogic.kubernetes.utils.FileUtils.checkDirectory;
+import static oracle.weblogic.kubernetes.utils.FileUtils.doesFileExist;
+
 
 /**
  *  Implementation of actions that download/install tools for the uses to use.
  *  NOTE: This class is a temporary solution, and may go away once we eventually
  *  install everything before the Java test starts to run.
  */
 
-public class Installer extends InstallWITCommon {
-  // temporary dir, will fix this once we decide on the
-  private static final String downloadDir = WORK_DIR + "/download";
+public class Installer {
+
   private InstallParams params;
+  
+  /**
+   * Set up the installer with default values
+   * @return the installer instance 
+   */
+  public static InstallParams defaultInstallParams() {
+    return new InstallParams().defaults();
+  }
 
   /**
    * Set up the installer with given parameters
    * @return the installer instance 
    */
-  public Installer with(InstallParams params) {
+  public static Installer withParams(InstallParams params) {
+    return new Installer().with(params);
+  }
+  
+  private Installer with(InstallParams params) {
```

Comment:
Fluent style would have this method just be named params

Reply:
I named it with() to indicate that this returns an Installer instance with some parameters. Name params() might give readers an impression that this returns a params instance. But I am new to fluent style and my understanding might be wrong.